src/components/z-offcanvas/styles.css

```diff
@@ -62,9 +39,14 @@
   right: 0;
   bottom: 0;
   left: 0;
+  display: flex;
```

Comment:
questa direi che non serve perché la prendiamo grazie a `:host([open])`

Reply:
serve perché nella variante overlay definiamo la posizione dell'elemento tramite flex `"justify-content"`, nel caso venga tolto il display alla chiusura `:not(open)` perde la posizione e la transazione di uscita parte dal lato opposto.